packages/docs-components/src/knobs/useSelectKnob.ts

```diff
@@ -1,9 +1,9 @@
 import { UseKnobOptions } from './types'
 import useKnob from './useKnob'
 
-const useSelectKnob = (options: UseKnobOptions<string>) =>
-  useKnob<string>({
-    initialValue: '',
+const useSelectKnob = <T = string>(options: UseKnobOptions<T>) =>
+  useKnob<T>({
+    initialValue: undefined,
```

Comment:
@layershifter this should be safe, right? is this still needed or can I omit?

Reply:
As I know it was changed in #1290, so you can just pick this change on merge